.github/workflows/add_to_project.yml

```diff
@@ -1,17 +0,0 @@
-name: Add To GitHub projects
-
-on:
-  workflow_dispatch:
-  issues:
-    types:
-      - opened
-
-jobs:
-  add-to-project:
-    name: Add issue to project
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/add-to-project@v0.5.0
-        with: 
-          project-url: https://github.com/orgs/metno/projects/12
-          github-token: ${{ secrets.ADD_TO_PROJECT_PAT }}
```

Comment:
Where is the secrets.ADD_TO_PROJECT_PAT set?
Why only issues and not also PRs?

Reply:
I removed it from the settings. The action was originally built to test on issues but didn't work, do I did not extend it to PRs.